db/migrate/20220508184141_rename_urgent_flag_to_flagged.rb

```diff
@@ -0,0 +1,8 @@
+class RenameUrgentFlagToFlagged < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :patients, :urgent_flag, :flagged
+    rename_column :archived_patients, :urgent_flag, :flagged
+
+    Patient.reset_column_information
```

Comment:
I'm not familiar with this pattern, can you say more about what we're trying to do here?